en/collect/import-using-online-bibliographic-database/slr.md

```diff
@@ -0,0 +1,59 @@
+# Conducting automated cross-library search
+
```

Comment:
It would be cool if you could describe the feature in one or two sentences. 
e.g. JabRef has the possibilty to do automatic cross library searches for systematic literature review...